cdk/lib/housekeeper.ts

```diff
@@ -0,0 +1,70 @@
+import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
+import { GuStack } from '@guardian/cdk/lib/constructs/core';
+import { GuLambdaFunction } from '@guardian/cdk/lib/constructs/lambda';
+import type { App } from 'aws-cdk-lib';
+import { PolicyStatement } from 'aws-cdk-lib/aws-iam';
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
+import { SnsEventSource } from 'aws-cdk-lib/aws-lambda-event-sources';
+import { Topic } from 'aws-cdk-lib/aws-sns';
+
+export class Housekeeper extends GuStack {
+	constructor(scope: App, id: string, props: GuStackProps) {
+		super(scope, id, props);
+
+		const bounceSNSTopic = new Topic(this, 'BounceSNSTopic', {
+			displayName:
+				'SNS topic alerted when AWS SES emails bounce (eg from Ophan Dashboard alerts, or Airflow)',
+			topicName: 'ses-email-bounce-notifications-for-housekeeper',
+		});
+		this.overrideLogicalId(bounceSNSTopic, {
+			logicalId: 'BounceSNSTopic',
+			reason: "We don't want AWS to delete the old topic and create a new one.",
+		});
+
+		const permanentEmailBounceTopic = new Topic(
+			this,
+			'PermanentEmailBounceTopic',
+			{
+				displayName: "Threat to email-sending ability of Ophan's AWS account",
+			},
+		);
+		this.overrideLogicalId(permanentEmailBounceTopic, {
+			logicalId: 'PermanentEmailBounceTopic',
+			reason: "We don't want AWS to delete the old topic and create a new one.",
+		});
+
+		const app = 'housekeeper';
+
+		const loggingPolicy = new PolicyStatement();
+		loggingPolicy.addResources('arn:aws:logs:*:*:*');
+		loggingPolicy.addActions(
+			'logs:CreateLogGroup',
+			'logs:CreateLogStream',
+			'logs:PutLogEvents',
+		);
```

Comment:
minor: this could a _little_ terser via the constructor props:

```ts
const loggingPolicy = new PolicyStatement({
  actions: [
    'logs:CreateLogGroup',
		'logs:CreateLogStream',
		'logs:PutLogEvents'
  ],
  resources: [
    'arn:aws:logs:*:*:*'
  ],
});
```

Reply:
I was thinking if there was a similar way of having those props inside but I wasn't sure. That looks better! Done the change 👍🏼